docs/kubernetes/domains-google-cloud-infrastructure.md

```diff
@@ -0,0 +1,53 @@
+# Domains in Google Cloud infrastructure.
+Every change in domain needs to be done manually. To pass traffic through domain name you need to modify `ingress.yml` and `webserver-php-fpm.yml` which serves as webserver and listens on domain names.
+
+## Adding Domain
+Let's say we want to listen on new domain name.
+
+Open [ingress.yml](/project-base/kubernetes/ingress.yml) file and add new domain block into `spec -> rules`:
+
+```
+// kubernetes/ingress.yml
+
+rules:
+    -   host: ~
+        http:
+            paths:
+            -   path: /
+                backend:
+                    serviceName: webserver-php-fpm
+                    servicePort: 8080
+```
+*Note*: ${DOMAIN_INDEX} is a index of domain in ingress.yml
```

Comment:
I am confused by this note - the variable is not used anywhere. 

Second thing, I would use aterisks for the whole note an use backticks :innocent:
```
*Note: `${DOMAIN_INDEX}` is a index of domain in `ingress.yml`*
```

Reply:
oh, or should it be "${DOMAIN_NUMBER}"?

Reply:
ye and under different code